src/main/resources/assets/create/lang/pl_pl.json

```diff
@@ -0,0 +1,1157 @@
+	"item.create.chromatic_compound": "Barwny związek",
+	"item.create.shadow_steel": "Cienista stal",
+	"item.create.blueprint_and_quill": "Schemat z piórem",
+	"item.create.blueprint": "Schemat",
+	"item.create.belt_connector": "Taśma mechaniczna",
+	"item.create.goggles": "Gogle inżynierskie",
+	"item.create.filter": "Filtr",
+	"item.create.property_filter": "Filtr atrybutów",
+	"item.create.rose_quartz": "Różowy kwarc",
+	"item.create.polished_rose_quartz": "Wypolerowany różowy kwarc",
+	"item.create.refined_radiance": "Rafinowany blaskowicz",
+	"item.create.iron_sheet": "Blaszka żelaza",
+	"item.create.gold_sheet": "Blaszka złota",
+	"item.create.lapis_plate": "Płytka lazurytu",
+	"item.create.obsidian_dust": "Sproszkowany obsidian",
+	"item.create.propeller": "Śmigło",
+	"item.create.whisk": "Trzepaczka",
+	"item.create.brass_hand": "Dłoń",
+	"item.create.slot_cover": "Klapka rzemieślnika mechanicznego",
+	"item.create.zinc_handle": "Cynkowa rękojeść",
+	"item.create.flour": "Mąką pszenna",
+	"item.create.dough": "Ciasto",
+	"item.create.wrench": "Klucz",
+	"item.create.deforester": "Mechaniczna siekiera",
+	"item.create.crushed_iron": "Zmielona ruda żelaza",
+	"item.create.crushed_gold": "Zmielona ruda złota",
+	"item.create.sand_paper": "Biały papier ścierny",
+	"item.create.red_sand_paper": "Czerwony papier ścierny",
+	
+	"item.create.brass_ingot": "Sztabka mosiądzu",
+	"item.create.brass_sheet": "Blaszka mosiądzu",
+	"item.create.brass_nugget": "Bryłka mosiądzu",
+	"item.create.crushed_brass": "Zmielony mosiądz",
+	"item.create.zinc_ingot": "Sztabka cynku",
+	"item.create.zinc_nugget": "Bryłka cynku",
+	"item.create.crushed_zinc": "Zmielona ruda cynku",
+	"item.create.copper_sheet": "Blaszka miedzi",
+	"item.create.copper_ingot": "Sztabka miedzi",
+	"item.create.copper_nugget": "Bryłka miedzi",
+	"item.create.crushed_copper": "Zmielona ruda miedzi",
+
+	"item.create.electron_tube": "Lampa elektronowa",
+	"item.create.integrated_circuit": "Układ scalony",
+	
+	"item.create.blazing_pickaxe": "Płomienny kilof",
+	"item.create.blazing_shovel": "Płomienna łopata",
+	"item.create.blazing_axe": "Płomienna siekiera",
+	"item.create.blazing_sword": "Płomienny tasak",
+
+	"item.create.shadow_steel_pickaxe": "Cienisty stalowy kilof",
+	"item.create.shadow_steel_mattock": "Cienista stalowa graca",
+	"item.create.shadow_steel_sword": "Cienisty stalowy miecz",
+
+	"item.create.rose_quartz_pickaxe": "Złocony kwarcowy kilof",
+	"item.create.rose_quartz_shovel": "Złocona kwarcowa łopata",
+	"item.create.rose_quartz_axe": "Złocona kwarcowa siekiera",
+	"item.create.rose_quartz_sword": "Złocone kwarcowe ostrze",
+
+	"block.create.copper_ore": "Ruda miedzi",
+	"block.create.copper_block": "Blok miedzi",
+	"block.create.copper_shingles": "Miedziane gonty",
+	"block.create.zinc_ore": "Ruda cynku",
+	"block.create.zinc_block": "Blok cynku",
+	"block.create.brass_block": "Blok mosiądzu",
+	
+	"block.create.andesite_casing": "Andezytowa skrzynia",
```

Comment:
For each of these casings, consider changing from _"skrzynia"_ to _"obudowa"_. In this translation, it will imply that it is equal to a chest which it is not.

Reply:
Good idea.